app/models/user/profile.rb

```diff
@@ -5,18 +5,25 @@ class User::Profile < ApplicationRecord
 
   delegate :to_param, to: :user
 
+  COUNT_TO_SHOW_TAB = 4
+
+  def self.count_to_show_tab
+    COUNT_TO_SHOW_TAB
+  end
+
   memoize
   def solutions_tab?
-    user.solutions.published.count > 3
+    user.solutions.published.count >= self.class.count_to_show_tab
   end
 
   memoize
   def testimonials_tab?
-    user.mentor_testimonials.published.count > 3
+    user.mentor_testimonials.published.count >= self.class.count_to_show_tab
```

Comment:
These two aren't the same as each other either. The fact that they're both 3 is a conicidence atm. So either use two constants, but I'd rather we didn't (as this class will quickly get messy) and instead just stub the method in the tests.